packages/stark-ui/.eslintrc.json

```diff
@@ -0,0 +1,41 @@
+{
+  "extends": "../../.eslintrc.json",
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
```

Comment:
This file could be simplified since the rules are already defined in the root file:

```json
{
  "extends": "../../.eslintrc.json",
  "ignorePatterns": ["!**/*"],
  "overrides": [
    {
      "files": ["*.spec.ts"],
      "rules": {
        "@angular-eslint/component-selector": "off",
        "@angular-eslint/directive-selector": "off",
        "@typescript-eslint/unbound-method": "off"
      }
    }
  ]
}
```